test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java

```diff
@@ -14,19 +14,6 @@
  **/
 package ddf.catalog.test;
 
-import org.junit.After;
```

Comment:
It doesn't look like you need this import reordering for this fix.  Can you remove TestCatalog from your commit?